e2e/articles.spec.ts

```diff
@@ -220,4 +220,52 @@ test.describe("Authenticated Articles Page", () => {
 
     await expect(page.getByText(commentContent)).toBeVisible();
   });
+
+  test.describe("Article sort order on article page", () => {
+    const sortOptions = ["newest", "oldest", "top"];
+
+    for (const sortOption of sortOptions) {
```

Comment:
Id recommend breaking ["newest", "oldest", "top"] into their own tests.

Yes we are repeating outselves but we also get test isolation ie. currently if this test breaks we will need to go into the actual test to see which option it breaks on

Reply:
Yeah, I was not sure what to do there. I will split it into separate tests. Thank John.

BTW is using the data attribute(for the like count ) the correct way to go about this?

Reply:
If your free to sync for 5 mins I can show you the way I get the most specific way to target an element.

TLDR if your not free. 

- Install playwright plugin for vs code
- Open the plugin view in vs code
- Use the pick locator function and hover over the element you want to target
- The plugin will give you the best locator to use

Reply:
@JohnAllenTech 
Great, I just hacked my way through the test today, without reading the docs 🫢 
I just downloaded that plugin, so will give it a try now.